tests/localnet/conftest.py

```diff
@@ -13,3 +15,57 @@ def algod_health_fast_timings(mocker: MockerFixture) -> None:  # noqa: ignore[PT
 @pytest.fixture()
 def health_success(httpx_mock: HTTPXMock) -> None:  # noqa: ignore[PT004]
     httpx_mock.add_response(url=ALGOD_HEALTH_URL)
+
+
+@pytest.fixture()
+def _localnet_up_to_date(proc_mock: ProcMock, httpx_mock: HTTPXMock) -> None:
```

Comment:
Minor detail, i think
`_localnet_up_to_date_` -> `localnet_up_to_date` might be better? 

Its used in at least 2 separate test sets so i assume it makes more sense to remove the `_`. If you look at the only similar example of 'private' fixture in `test_doctor.py` that fixture is only used within a scope of a single file hence makes sense to mark it private there.

Reply:
I added the underscore because of this error 
Fixture `localnet_up_to_date` does not return anything, add leading underscoreRuff(PT004)

Reply:
I see, looks like ruff is enforcing pytest conventions, ignore this one then all good